### PR TITLE
Parallax Welcome screen animation

### PR DESCRIPTION
**Pull request by kaitoo1**

### CODE REVIEW

File: src/flows/OnboardingFlow/steps/Welcome.tsx [hunk 1]
```diff
@@ -5,8 +5,18 @@ import Button from 'components/core/Button/Button';
 import colors from 'components/core/colors';
 import Spacer from 'components/core/Spacer/Spacer';
 import FullPageCenteredStep from 'flows/shared/components/FullPageCenteredStep/FullPageCenteredStep';
+import CoolIntro from 'scenes/CoolIntro/CoolIntro';
 
 function Welcome({ next }: WizardContext) {
+  // return <WelcomeText next={next} />;
+  return <CoolIntro next={next}></CoolIntro>;
```
Comment (fbrcsftnr) [inline src/flows/OnboardingFlow/steps/Welcome.tsx:12]: fix?

File: src/scenes/CoolIntro/CoolIntro.tsx [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+import Button from 'components/core/Button/Button';
+import colors from 'components/core/colors';
+import Spacer from 'components/core/Spacer/Spacer';
+
+import styled, { css, keyframes } from 'styled-components';
+import { useSpring, animated } from 'react-spring';
+import './intro.css';
+import { useEffect } from 'react';
+
+const calc = (x: number, y: number) => [
```
Comment (fbrcsftnr) [inline src/scenes/CoolIntro/CoolIntro.tsx:12]: could we add a comment explaining what this does?

File: src/scenes/CoolIntro/CoolIntro.tsx [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+  x - window.innerWidth / 2,
+  y - window.innerHeight / 2,
+];
+type AnimatedImage = {
+  src?: string;
+  width: number;
+  // zIndex is the "depth" of image from viewer's pov.
+  // It is used to calculate how the image will move based on mousemovement.
+  // z-index of 0 is essentially the axis of movement. positive values move in one direction, negative values move in opposite.
+  // values can be -100 to 100, with -100 being the furthest from viewer and 100 being closest.
```
Comment (fbrcsftnr) [inline src/scenes/CoolIntro/CoolIntro.tsx:22]: beautiful

File: src/flows/OnboardingFlow/steps/Welcome.tsx (hunk 1 above)
Comment (kaitoo1) [reply src/flows/OnboardingFlow/steps/Welcome.tsx:12]: Done - removed commented line and renamed CoolIntro to WelcomeAnimation lol

**Pull request merged**